.isort.cfg

```diff
@@ -2,12 +2,13 @@
 ; see https://github.com/psf/black
 multi_line_output=3
 include_trailing_comma=True
-force_grid_wrap=0
+force_grid_wrap=2
 combine_as_imports=True
 use_parentheses=True
-line_length=110
```

Comment:
Please revert those changes. All the repos use the same config.

Reply:
Yeah I'll revert this, seems that this does not help to fix the problem with black and isort. That's why the fix I put was to put the skip on the import.